app/scripts/modules/core/src/pipeline/config/stages/monitoreddeploy/DeploymentMonitorExecutionDetails.tsx

```diff
@@ -108,17 +111,23 @@ export class DeploymentMonitorExecutionDetails extends React.Component<
             <dl className="dl-narrow dl-horizontal">
               <dt>Summary</dt>
               <dd>{getDeploymentMonitorSummary(stage)}</dd>
-              <dt className="sp-margin-l-bottom">Deploy %</dt>
-              <dd className="sp-margin-l-bottom">
-                <span>{stage.context.currentProgress}%</span>
-              </dd>
-              <dt>Details</dt>
-              <dd>{details || notProvidedFragment}</dd>
-              <dt className="sp-margin-l-bottom">Log</dt>
-              <dd className="sp-margin-l-bottom">{log ? <pre>{log}</pre> : notProvidedFragment}</dd>
-              <dt>More info</dt>
-              <dd>{additionalDetails ? '' : notProvidedFragment}</dd>
-              {additionalDetails}
+              {stage.hasNotStarted ? (
```

Comment:
why the ternary with an empty string vs. `stage.hasNotStarted && (....)`?

Reply:
or... maybe we want to add an exact inverse method (`stage.hasStarted`?)

Reply:
i thought the ternary was easier to read, if not started, show nothing, otherwise ...

vs. if not not started show something. i am cool either way

Reply:
🤷‍♂ seems reasonable. only nitpicky suggestion would be to switch your usage of `''` here and on L127 to use `null` instead (wading into super nit territory, null is usually more idiomatic in JSX for 'I don't want to render anything here')

Reply:
gotcha, done